manager/apis/v2/handler/scheduler_instance.go

```diff
@@ -0,0 +1,203 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+
+	"d7y.io/dragonfly/v2/manager/apis/v2/types"
+	"d7y.io/dragonfly/v2/manager/store"
+	"d7y.io/dragonfly/v2/pkg/dfcodes"
+	"d7y.io/dragonfly/v2/pkg/dferrors"
+	"github.com/gin-gonic/gin"
+	"gopkg.in/errgo.v2/fmt/errors"
+)
+
+// AddSchedulerInstance godoc
+// @Summary Add scheduler instance
+// @Description add by json config
+// @Tags SchedulerInstance
+// @Accept  json
+// @Produce  json
+// @Param instance body types.SchedulerInstance true "Scheduler instance"
+// @Success 200 {object} types.SchedulerInstance
+// @Failure 400 {object} HTTPError
+// @Failure 404 {object} HTTPError
+// @Failure 500 {object} HTTPError
+// @Router /schedulerinstances [post]
+func (handler *Handler) AddSchedulerInstance(ctx *gin.Context) {
```

Comment:
Rename `AddSchedulerInstance` to `CreateScheduler`.
RESTful API: ![image](https://user-images.githubusercontent.com/15955374/117446103-2d86bb80-af6e-11eb-9cab-7656b68d1fca.png)